.travis.yml

```diff
@@ -1,7 +1,7 @@
 # We only use Python for coveralls; both Python 2.7 and 3.6 are
```

Comment:
comment for python 3.7 here. Though not that it really matters since no travis CI is running here.